### PR TITLE
translate api search resource

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Translate API search resource

**Which issue(s) this PR fixes**:
Part of #483

**Special notes for your reviewer**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/search-resources/resource-registry-v1alpha1.md [hunk 1]
```diff
@@ -32,151 +32,151 @@ ResourceRegistry represents the configuration of the cache scope, mainly describ
...
+  - **conditions.message**（string），必选
 
-    message is a human readable message indicating details about the transition. This may be an empty string.
+    message 是有关转换的详细信息（人类可读消息）。可以是空字符串。
 
-  - **conditions.reason** (string), required
+  - **conditions.reason**（string），必选
 
-    reason contains a programmatic identifier indicating the reason for the condition's last transition. Producers of specific condition types may define expected values and meanings for this field, and whether the values are considered a guaranteed API. The value should be a CamelCase string. This field may not be empty.
+    reason 是一个程序标识符，表明状况最后一次转换的原因。特定状况类型的生产者可以定义该字段的预期值和含义，以及这些值是否可被视为有保证的 API。取值应该是一个 CamelCase 字符串。此字段不能为空。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/search-resources/resource-registry-v1alpha1.md:163]:
```suggestion
    reason 是一个程序标识符，表明状况最后一次转换的原因。特定状况类型的生产者可以定义该字段的预期值和含义，以及这些值是否可被视为有保证的 API。取值应该是一个驼峰式（CamelCase）的字符串。此字段不能为空。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/search-resources/resource-registry-v1alpha1.md [hunk 2]
```diff
@@ -32,151 +32,151 @@ ResourceRegistry represents the configuration of the cache scope, mainly describ
...
 
-  - **conditions.status** (string), required
+  - **conditions.status**（string），必选
 
-    status of the condition, one of True, False, Unknown.
+    status 表示状况的状态。取值为True、False或Unknown。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/search-resources/resource-registry-v1alpha1.md:167]:
```suggestion
    status 表示状况的状态。取值为 True、False 或 Unknown。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/search-resources/resource-registry-v1alpha1.md (hunk 1 above)
Comment (XiShanYongYe-Chang) [reply i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/search-resources/resource-registry-v1alpha1.md]: 已修改，感谢~

File: i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/search-resources/resource-registry-v1alpha1.md (hunk 2 above)
Comment (XiShanYongYe-Chang) [reply i18n/zh/docusaurus-plugin-content-docs/current/reference/karmada-api/search-resources/resource-registry-v1alpha1.md]: 已修改，感谢~

**Review by RainbowMango — APPROVED** [review]
/approve
Thanks @windsonsea

**Pull request merged**